node-rfc/README.md

```diff
@@ -0,0 +1,63 @@
+# RFC CTS+ Docker File
+## Description
+To bring your SAP Fiori application to your ABAP Front End Server you always need to create a transport request and upload the zip file to it. If you want to automate your delivery pipeline for this scenario, and you have your NW ABAP < 7.50 SP08, you can only do it by using RFC communication. But to get it run you need to do some installation and configuration steps.
+A detailed scenario description can be found in our [best practice guid][bestpractice].
+
+This Dockerfile provides an easy and consumable way for the node-rfc wrapper and the SAP NetWeaver RFC Library. The only thing you need to have is a container environment like docker to build and to execute the node-rfc wrapper image. You can use it standalone in your CI/CD environment or you can use it within our Jenkins shared library [project "Piper"][piper].
+
+## Requirements
+
+* General requirements can be found at the [repository readme][general]
+* An S-User for [SAP ONE][sapone]
+* Please Download SAPCAR for Linux on x86_64 from [SAP ONE][sapone]
+* Please Download NWRFC library ```SAP NW RFC SDK 7.50``` for Linux on x86_64 from [SAP ONE][sapone]
+* A local docker image repository
```

Comment:
I think this can be omitted. This is already there, given you have a Docker environment like the one mentioned in general requirements.